### ISSUE TITLE
Prac 4, Task 5.2

### ISSUE THREAD

**Issue by ZacharyP-E** [jreades/fsds#30]
Prac 4, Task 5.2

!touch dtools/__init__.py
# Comment out so we don't automatically re-run this code every time
!jupyter nbconvert --ClearOutputPreprocessor.enabled=True \
    --to python --output=dtools/notebook.py \
    Practical-04-Objects.ipynb

Instead of  Practical-04-Objects-Answers.ipynb

**Comment by jreades** [comment]
Good call. Didn’t realise I’d not done this. Going to actually move this to week 8 on text now.
